### ISSUE TITLE
Allow multiple commit messages to be skipped

### ISSUE THREAD

**Issue by paced** [veggiemonk/skip-commit#3]
It would be helpful to be able to run `skip-ci` but also maybe `automated` (for example) to skip a build. It allows the commit message to provide context as well as skipping the workflow.

**Comment by veggiemonk** [comment]
Hi @paced
You can specify that by using the environment variable `COMMIT_FILTER`
See for code: https://github.com/veggiemonk/skip-commit/blob/master/entrypoint.sh#L10

**Comment by paced** [comment]
Okay, in that case, perhaps the documentation should be updated to show that you can use a regular expression for `COMMIT_FILTER`?

**Comment by veggiemonk** [comment]
@paced I would not use the container given the latest update to the github action.

Now, it is possible to use a bash command directly. It is faster.

see https://github.com/veggiemonk/skip-commit#just-use-bash

Do you really need the container? 
If it is the case, you can use a different step for each word that needs skipping.

**Comment by paced** [comment]
Calling `bash` directly makes this a custom workflow step and not an Action. I expect GitHub will add custom languages (shellscript, Python and so on) in the future but right now, it's only possible with JavaScript.

It is still possible to just use bash and use a regular expression for `COMMIT_FILTER`, so adding that to the documentation would improve usability, I think.

I've opened a fork with these changes. I've decided not to open a pull request since it's quite a large departure from the code found in this repository: [teaminkling/skip-commit](https://github.com/teaminkling/skip-commit).

Thanks!
